reviews/serializers.py

```diff
@@ -41,6 +41,32 @@ def __repr__(self):
     return IfCanViewLogCreatorField
 
 
+class ReviewableCountingRelationshipField(RelationshipField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs['related_meta'] = kwargs.get('related_meta') or {}
+        kwargs['related_meta']['include_status_counts'] = True
+        super(ReviewableCountingRelationshipField, self).__init__(*args, **kwargs)
+
+    def get_meta_information(self, metadata, provider):
+        metadata = metadata or {}
+
+        # Make counts opt in
+        show_counts = utils.is_truthy(self.context['request'].query_params.get('related_counts', False))
+        # Only include counts on detail routes
+        is_detail = not isinstance(self.context['view'], generics.ListAPIView)
+        # Weird hack to avoid being called twice
+        is_related_meta = metadata.get('include_status_counts')
+
+        if show_counts and is_detail and is_related_meta:
+            # Finally, require users to have view_review_logs permissions
+            auth = utils.get_user_auth(self.context['request'])
+            if auth and auth.logged_in and auth.user.has_perm('view_review_logs', provider):
+                metadata = provider.get_reviewable_status_counts()
```

Comment:
Will this overwrite existing metadata?

Reply:
It will but there currently isn't any. I'll update it.